go/vt/topo/etcd2topo/server.go

```diff
@@ -87,6 +113,12 @@ func NewServer(serverAddr, root string) (*Server, error) {
 	}, nil
 }
 
+// TODO: Rename this to a name to signifies this function uses the process-wide TLS settings.
```

Comment:
The `NewServer ...` part of the comment needs to be the first line of the comment block. That's a special convention required by the godoc generator.

Reply:
Done.